src/main/java/frc/robot/subsystems/Elevator/ElevatorSubsystem.java

```diff
@@ -45,41 +40,16 @@ public class ElevatorSubsystem extends SubsystemBase {
               Math.toDegrees(Constants.ElevatorConstants.elevatorAngleRad),
               15,
               new Color8Bit(Color.kPurple)));
-  MechanismLigament2d armLig2d =
-      elevatorLig2d.append(
-          new MechanismLigament2d(
-              "Arm",
-              Constants.ArmConstants.armLengthInches,
-              90,
-              15,
-              new Color8Bit(Color.kLavender)));
 
   public ElevatorSubsystem() {
-    elevatorMotor = new HighlanderFalcon(Constants.ElevatorConstants.elevatorMotorID, 5.45 / 1.0);
-    elevatorMotor.config_kF(0, 0);
-    elevatorFollower = new HighlanderFalcon(Constants.ElevatorConstants.elevatorFollowerID);
-    elevatorFollower.set(ControlMode.Follower, Constants.ElevatorConstants.elevatorMotorID);
-    elevatorFollower.configSupplyCurrentLimit(
-        new SupplyCurrentLimitConfiguration(true, 30.0, 30.0, 0.5));
-    elevatorMotor.configSupplyCurrentLimit(
-        new SupplyCurrentLimitConfiguration(true, 30.0, 30.0, 0.5));
-    elevatorMotor.configVoltageCompSaturation(10);
-    elevatorFollower.configVoltageCompSaturation(10);
-    LoggingWrapper.shared.add("elevatorsim", mech2d);
-    zeroMotor();
+    io = Robot.isReal() ? new ElevatorIOFalcon() : new ElevatorIOSim();
```

Comment:
nitpick: recommend using this. for member variables. Means I don't go looking for where it's defined.

Reply:
Also, it's going to be worth considering as you add more Sim versions (assuming you do that) whether that check (`Robot.isReal`) should be hoisted up a level and you have a Sim subsystem vs each subsystem having to know which parts are sim'd out.